indico/web/templates/forms/protection_widget.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html -%}
+    {{ field.radio_widget(field, **input_args)  }}
+    {{ field.render_protection_message() }}
+{%- endblock %}
+
+{% block javascript -%}
+    <script>
+        (function() {
+            'use strict';
+
+            var inputs = $('input[name={{ field.name }}][id^={{ field.id }}]');
+
```

Comment:
var parentProtected = {{ field.get_form().protected_object.protection_parent.is_protected | tojson }};
and then down in the change handler:

     var isProtected = $this.val() === 'protected' || ($this.val() === 'inheriting' && parentProtected);